<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogItem</class>
 <widget class="QDialog" name="DialogItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="labelFats">
       <property name="text">
        <string>Fats</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelCarb">
       <property name="text">
        <string>Carb</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCalories"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelFiber">
       <property name="text">
        <string>Fiber</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditFats"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCalories">
       <property name="text">
        <string>Calories</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditFiber"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditCarb"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelProtein">
       <property name="text">
        <string>Protein</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditProtein"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
